### ISSUE TITLE
Unnecessary prefix is appended to Port Already in use error

### ISSUE THREAD

**Issue by chamil321** [ballerina-platform/ballerina-lang#17296]
**Description:**
$ Title
`error: invoking main method failed due to error {ballerina/http}ListenerStartupError message=failed to start server connector '0.0.0.0:9090': Address already in use cause=null`

The `error: invoking main method failed due to` prefix is not related to services.

**Comment by warunalakshitha** [comment]
Latest pack i am getting following.

Exception in thread "main" error {ballerina/http}ListenerStartupError message=failed to start server connector '0.0.0.0:9090': Address already in use cause=null
	at org.ballerinalang.jvm.BallerinaErrors.createError(BallerinaErrors.java:67)
	at org.ballerinalang.net.http.HttpUtil.createHttpError(HttpUtil.java:517)
	at org.ballerinalang.net.http.serviceendpoint.Start.startServerConnector(Start.java:85)
	at org.ballerinalang.net.http.serviceendpoint.Start.start(Start.java:59)
	at ballerina.http.Listener.start(service_endpoint.bal:86)
	at ballerina.http.Listener.call(service_endpoint.bal)
	at ballerina.http.Listener.__start(service_endpoint.bal:37)
	at ballerina.http.Listener.call(service_endpoint.bal)
	at ___init.___start_(.:29)
	at ___init.$lambda$___start_$(.)
	at org.ballerinalang.jvm.scheduling.SchedulerItem.execute(Scheduler.java:411)
	at org.ballerinalang.jvm.scheduling.Scheduler.run(Scheduler.java:198)
	at org.ballerinalang.jvm.scheduling.Scheduler.runSafely(Scheduler.java:169)
	at java.lang.Thread.run(Thread.java:748)

**Comment by SupunS** [comment]
Fixed on v1.0-beta-SNAPSHOT, and below error is logged:
```
error: {ballerina/http}ListenerStartupError message=failed to start server connector '0.0.0.0:8080': Address already in use cause=null
	at ballerina.http.$value$Listener:start(service_endpoint.bal:90)
	   ballerina.http.$value$Listener:__start(service_endpoint.bal:37)
	   ___init:..<init>(.:4)
	   ___init:..<init>(.:65535)
[ballerina/http] stopped HTTP/WS listener 0.0.0.0:8080
```

However still there are things to be cleaned up. For eg: `cause=null`, `$value$Listener`, `___init:..<init>(.:65535)`
